packages/core/src/amazonqGumby/errors.ts

```diff
@@ -12,6 +12,12 @@ export class JavaHomeNotSetError extends ToolkitError {
     }
 }
 
+export class NoAuthError extends ToolkitError {
+    constructor() {
+        super('Not authenticated', { code: 'AuthFailed' })
```

Comment:
i am certain we have existing ToolkitErrors for this concept. 

how many of these other custom errors are redundant?

Reply:
None of the other custom errors are redundant as they are highly specific to `/transform`, but you're right that we do have this: https://github.com/aws/aws-toolkit-vscode/blob/0f30f193be0ecdadd929b815c36ea3940cda02c9/packages/core/src/auth/sso/server.ts#L40

which I don't mind using, although it seems specific to redirect errors?